README.md

```diff
@@ -1 +1,84 @@
-# My-to-do-list
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My Todo-List] <a name="about-project"></a>
+
+**[My Todo-List]** A mini and simple Todo list website where users can add a todo-list and remove list from their collection.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS, JavaScript with webpack and linter files
+
+### Key Features <a name="key-features"></a>
+
+- **[linter files(debugging tools)]**
+- **[ html, JavaScript and css template]**
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Install git on your computer
+
+-code editor
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+-git@github.com:Mylo16/To-do-list.git
+
+### Deployment
+
+N/A
+
+
+## 👥 Author <a name="authors"></a>
+
+
+- GitHub: [@Mylo16](https://github.com/Mylo16)
```

Comment:
- [ ] I would advise you to give more contact of yours so that a recruiter can easily find you. Here is the design provided by Microverse:

👤 Author: Your Name

GitHub: [@githubhandle](https://github.com/githubhandle)
Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)